subsys/dfu/dfu_target/Kconfig

```diff
@@ -117,14 +117,37 @@ config DFU_TARGET_SUIT
 	bool "SUIT DFU target support"
 	default y
 	depends on SUIT
+	depends on SDFW_SERVICES_ENABLED
 	imply MPU_ALLOW_FLASH_WRITE
 	select SUIT_UTILS
 	select ZCBOR
 	select ZCBOR_CANONICAL
-	select DFU_TARGET_STREAM
+	imply FLASH
+	imply STREAM_FLASH
+	imply STREAM_FLASH_ERASE
+	imply DFU_TARGET_STREAM
+	imply SUIT_DFU_CACHE_EXTRACT_IMAGE
+	imply SUIT_DFU
 	help
 	  Enable support for updates using DFU target based on SUIT
 
+if DFU_TARGET_SUIT
+
+# The full processing level is needed to process the SUIT manifest in DFU Target
```

Comment:
If this config is needed for this particular case, shouldn't we `select` it for the `DFU_TARGET_SUIT`?